prettier.config.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
   semi: false,
   singleQuote: true,
   trailingComma: 'all',
-  plugins: ['.'],
+  plugins: [require('.')],
```

Comment:
Is this broken?

Reply:
I guess that this change improves compatibility with PnP mode. It’s probably not needed otherwise.

Reply:
I'll look into it.

Reply:
It's bug in prettier v3.

Reply:
Or maybe not..

> Since v3.0.0, we load plugins via import() instead of require(), plugins can be ESM modules now.
> If you use --plugin by directory path, or file path without extensions, the plugin may not able to load.

https://github.com/prettier/prettier/blob/main/changelog_unreleased/api/13201.md

I forgot that.

Since you can't `import(".")`, but you can `import("./lib/index.js")`, and use `"./lib/index.js"` work as expected.